heartbeat/monitors/plugin_test.go

```diff
@@ -194,43 +192,6 @@ func Test_pluginsReg_String(t *testing.T) {
 	}
 }
 
-func Test_pluginBuilder_create(t *testing.T) {
```

Comment:
What happened to these?

Reply:
This whole file should be deleted. It was auto-generated, but never filled in. This particular test no longer compiled, so I just deleted it. I'll delete the whole file in a subsequent PR.